cmd/install/uninstall.go

```diff
@@ -85,6 +86,7 @@ func NewUninstallCmd() *cobra.Command {
 	cmd.AddCommand(NewUninstallStorageCmd())
 	cmd.AddCommand(NewUninstallAliasCmds(storageApps, "storage-")...)
 	cmd.AddCommand(velero.NewApp().NewUninstallCmd())
+	cmd.AddCommand(consul.NewApp().NewUninstallCmd())
```

Comment:
Would you mind putting this right after coredumphandler to keep it alphabetical?

Reply:
Fixed ordering